### PR TITLE
PR: Last Stretch for RC1

### PR DESCRIPTION
**Pull request by KelSolaar**
Last stretch PR for the release candidate 1.

### CODE REVIEW

**Review by doug-walker — DISMISSED** [review]
LGTM

You may want to update the docs/usage.rst to include the Studio command.  (And minor but ideally, the first "Task | Command" line in each table in usage.rst would be bold to distinguish it as the table header.)

**Comment by KelSolaar** [comment]
From #73 by @doug-walker:

> Awesome, thank you Thomas!
> 
> There are still a few issues with aliases, but this fixed the majority of them. The remaining ones are:
> 
> * "ACES" prefix missing from legacy name of first 9 spaces in the spreadsheet.
> * "Utility" prefix missing from legacy name of Raw color space.
> * P3-D65 alias has a dash (unlike the 1.2 config)
> * The "sRGB - Texture" color space has an alias "srgb" which I worry is too short and could conflict with other usage.  The 1.2 config did not have this as a name.  I don't know where this is coming from since it is not in the "Aliases" column of the spreadsheet.  Is the slugify dropping "texture"?

**Comment by KelSolaar** [comment]
> "ACES" prefix missing from legacy name of first 9 spaces in the spreadsheet.

The spreadsheet as of yesterday should be correct afaik.

> "Utility" prefix missing from legacy name of Raw color space.

`Raw` and `ACES2065-1` are implicit and manually created, I will add their aliases.

> P3-D65 alias has a dash (unlike the 1.2 config)

This should be addressed in this PR.

> The "sRGB - Texture" color space has an alias "srgb" which I worry is too short and could conflict with other usage. The 1.2 config did not have this as a name. I don't know where this is coming from since it is not in the "Aliases" column of the spreadsheet. Is the slugify dropping "texture"?

Yes the automatic alias generation process does drop `- Texture`, we can make a special case for `sRGB` here, what alias name would you think is best?

File: docs/opencolorio_config_aces.clf.rst [hunk 1]
```diff
@@ -43,6 +43,5 @@ Common LUT Format Generation
     generate_clf_transforms_davinci
```
[the remark this answers is not in the document]
Comment (KelSolaar) [reply docs/opencolorio_config_aces.clf.rst:43]: > LGTM
> 
> You may want to update the docs/usage.rst to include the Studio command. (And minor but ideally, the first "Task | Command" line in each table in usage.rst would be bold to distinguish it as the table header.)

The studio config tasks are documented but the section header is wrong and called `Reference Config` 🤦

**Review by doug-walker — APPROVED** [review]
Thank you Thomas, this addressed my notes!

**Comment by KelSolaar** [comment]
Awesome! Let me push a last commit to bump the version and we can do a release!

**Comment by KelSolaar** [comment]
Should be good now @doug-walker!

**Review by doug-walker — APPROVED** [review]
Whoop!

**Pull request merged**